src/components/application_manager/test/commands/hmi/simple_requests_to_hmi_test.cc

```diff
@@ -131,12 +141,12 @@ TEST_F(RequestToHMITest, SendRequest_SUCCESS) {
   SharedPtr<commands::RequestToHMI> command(
       CreateCommand<commands::RequestToHMI>());
 
-  EXPECT_CALL(mock_app_manager_, SendMessageToHMI(NotNull()));
+  EXPECT_CALL(app_mngr_, SendMessageToHMI(NotNull()));
 
   command->SendRequest();
 }
 
-template <typename Command>
+template <class Command>
```

Comment:
@wolfylambova22 Why did you change it? We don't use `<class ...>` in templates. Please read carefully: [SDL Coding Style](https://github.com/smartdevicelink/sdl_core/wiki/SDL-Coding-Style-Guide) and make appropriate changes.